### PR TITLE
Fix checkout refreshing

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it adds changes introduced in https://github.com/mirumee/saleor-sdk/pull/26. Fixes https://github.com/mirumee/saleor-storefront/issues/853

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Review by krzysztofwolski — APPROVED** [review]

File: src/@next/components/molecules/Thumbnail/types.ts [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+  alt: string | null;
+}
+
+interface IThumbnail2x {
+  url: string;
+}
+
+interface ISource {
+  thumbnail?: IThumbnail | null;
+  thumbnail2x?: IThumbnail2x | null;
```
Comment (dominik-zeglen) [inline src/@next/components/molecules/Thumbnail/types.ts:12]: Can't we just use some fragment type from API?

File: src/@next/components/organisms/ProductList/types.ts [hunk 2]
```diff
@@ -1,9 +1,7 @@
 import { ProductList_products_edges_node } from "@saleor/sdk/lib/queries/gqlTypes/ProductList";
 
-type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
-
 export interface IProps {
-  products: Array<PartialBy<ProductList_products_edges_node, "category">>;
+  products: Array<Partial<ProductList_products_edges_node>>;
```
Comment (dominik-zeglen) [inline src/@next/components/organisms/ProductList/types.ts:4]: I don't get why did you make this change, is it some part of code cleanup?

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

File: src/@next/components/organisms/ProductList/types.ts (hunk 2 above)
Comment (orzechdev) [reply src/@next/components/organisms/ProductList/types.ts]: `ProductList_products_edges_node` doesn't contain `category` anymore.

File: src/@next/components/organisms/ProductList/types.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { ProductList_products_edges_node } from "@saleor/sdk/lib/queries/gqlTypes/ProductList";
 
 export interface IProps {
-  products: Array<Partial<ProductList_products_edges_node>>;
+  products: Array<ProductList_products_edges_node>;
```
Comment (dominik-zeglen) [inline src/@next/components/organisms/ProductList/types.ts:4]: More readable

```suggestion
  products: ProductList_products_edges_node[];
```

**Review by dominik-zeglen — APPROVED** [review]

**Pull request merged**